notebooks/notebook.ipynb

```diff
@@ -1,5 +1,12 @@
 {
```

Comment:
<p>Perhaps the model parameters could be explained here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/repro-challenge-team-3/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>OK!</p>

Reply:
closing conversation, resolved